app/classification.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const teacherArray = ['taught','coached', 'trained', 'educated'];
+const studentArray = ['learned from'];
+const wroteArray = ['wrote','created', 'composed'];
+const playedArray = ['played','performed'];
+
+const dict = {teacher:teacherArray, student:studentArray, wrote:wroteArray, played:playedArray};
+
+module.exports.findRelationshipClass = function(word) {
+  for(let key in dict) {
+    let list = dict[key];
+    for(let w in list) {
+      if(list[w] === word) {
+        return key;
+      }
+    }
+  }
+  return null;
```

Comment:
I'm adding some new features as well :)
But I was thinking its better to know that the result is in fact a "class" or not, if you send "teacher" and get teacher back, how do you know if that is in fact the classification or simply that it could not classify it.